google/cloud/documentai_toolbox/wrappers/page.py

```diff
@@ -95,22 +112,37 @@ class FormField:
     _page: "Page" = dataclasses.field(repr=False)
 
     @cached_property
-    def field_name(self):
+    def field_name(self) -> str:
         return _trim_text(
             _text_from_layout(
                 self.documentai_object.field_name, self._page._document_text
             )
         )
 
     @cached_property
-    def field_value(self):
+    def field_value(self) -> str:
         return _trim_text(
             _text_from_layout(
                 self.documentai_object.field_value, self._page._document_text
             )
         )
 
 
+def _trim_text(text: str) -> str:
+    """Remove extra space characters from text (blank, newline, tab, etc.)
+
+    Args:
+        text (str):
+            Required. UTF-8 encoded text in reading order
+            from the document.
+
+    Returns:
+        str:
+            Text without trailing spaces/newlines
+    """
+    return text.strip().replace("\n", " ")
```

Comment:
Let's document this behavior in the docstring, it is not only removing space characters.

On the other hand, how can we be sure that the user does not car about leading/trailing white spaces (removed by `strip`) or newline characters in the middle of the text.

Reply:
It's true, we don't know for sure that the user needs these removed. It was added as a convenience because many form field names/values are misshapen when output from DocAI.

Should this be removed?